docs/projects/P2/2_firstsprint.md

```diff
@@ -0,0 +1,97 @@
+# Project 2B: Collaborative Development
+
+## Deliverables
+
+**First Sprint** – 100 points – due Tuesday, February 14th, 11:59pm
+
+- [Process & Implementation Interim Snapshot](#process--implementation-interim-50-pts) (50 pts)
+- [Checkpoint Presentation](#checkpoint-presentation-50-pts) (50 pts) - **Held in Recitation on Monday, February 13th**
+
+
+## Process & Implementation Interim (50 pts)
+
+Start by cloning the team repository and merging your individual changes from Project 1 into your team repository using proper git flow. You may also delete the `hw1.yaml` workflow found in `.github/workflows`. Once everyone has properly set up the repository on their local machines and committed their changes, you can now start development.
+
+You should follow best practices for collaborative development such as using feature branches, pull requests, and code review. Individual changes should be committed to properly named branches for each core feature being implemented and make use of pull requests to merge changes together. 
+
+Be sure to **review each other’s code** - both from a quality assurance standpoint, but also so you have a better understanding of the codebase. Although code reviews can be done asynchronously, we encourage the first few to be done synchronously so you can discuss the changes with your teammates and learn more about the codebase.
+
+To fit with the translation efforts described in Project 1, any new code introduced to the codebase should be in TypeScript. This does mean that you may need to translate existing files to fulfill this requirement.
```

Comment:
I'm wondering if we should still be strict about this given the current understanding of how they are with in Project 1?